src/client/java/teammates/client/scripts/AddSectionsToLargeCourses.java

```diff
@@ -7,9 +7,9 @@
 import java.util.Set;
 
 import teammates.client.remoteapi.RemoteApiClient;
-import teammates.common.datatransfer.CourseDetailsBundle;
-import teammates.common.datatransfer.StudentAttributes;
-import teammates.common.datatransfer.TeamDetailsBundle;
+import teammates.common.datatransfer.attributes.CourseDetailsBundle;
```

Comment:
`*Bundle` should not be under `attributes` sub-package. This sub-package should be strictly for `*Attributes`.